index.html

```diff
@@ -311,11 +320,15 @@ <h2 id="contact-form-text">
                       </h2>
                           <div id="form-field-container">
                           <input type="text" name="fname" id="fname" placeholder="First Name" required maxlength="30">
+                          <span class="errorMsg"></span>
                           <input type="text" name="lname" id="lname" placeholder="Last Name" required maxlength="30">
+                          <span class="errorMsg"></span>
                           <input type="email"  name="email-field" id="email-field" placeholder="Email address" required maxlength="30">
+                          <span class="errorMsg"></span>
```

Comment:
- [ ] Kindly be noticed that the error message should show up close to the "submit" button.

Reply:
Done as advised